src/App.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-env browser */
+import Layout from "@organisms/Layout"
+import React, { Suspense } from "react"
+import { useTranslation } from "react-i18next"
+import { Route, Routes } from "react-router-dom"
+import Home from "./pages"
+import NotFound from "./pages/NotFound"
+import About from "./pages/about"
+import Events from "./pages/events"
+import Projects from "./pages/projects"
+
+const App = () => {
+  const { i18n } = useTranslation()
+  const baseUrl = i18n.language === "it" ? "" : "/" + i18n.language
+
+  return (
+    <Suspense fallback="loading">
+      <Routes>
+        <Route element={<Layout />}>
+          <Route path={`/${baseUrl}`} element={<Home />} />
+          <Route path={`/${baseUrl}/about`} element={<About />} />
+          <Route path={`/${baseUrl}/projects`} element={<Projects />} />
+          <Route path={`/${baseUrl}/events`} element={<Events />} />
```

Comment:
The use of `baseUrl` in this way it's very dangerous, and in this way would very likely break the current deployment (reminder for me that I should add more tests about that...).

At the very least before the `/$(baseUrl)` there should be a `.`, to ensure relative paths still works, 

A better way to handle this would be using [React Router's params](https://reactrouter.com/en/main/hooks/use-params). They used to support also optional parameters, so we could set a default language, however this is not the case [since v6](https://github.com/remix-run/react-router/issues/7285), but a couple of workarounds are given [here](https://stackoverflow.com/a/70961421/9226023).

Reply:
From version 6.5 have been reintroduced, only, I don’t know why, it gave me problems with the home, which was not rendered in Italian, so I decided to try this solution that seemed to work

Reply:
I admit I haven't actually tested if it worked or not in the case a different `BASE_PATH`, but by just reading the code it looks likely.

In case you too want to test in the future, you can do so by executing `set BASE_PATH=/isaac-website` before running `npm start dev`, that should deploy the website in the subfolder in Windows. To unset the variable just `set BASE_PATH=`, or just close and reopen the terminal.